content/docs/security/external-auth.md

```diff
@@ -0,0 +1,323 @@
+---
+title: External auth
+weight: 10
+description: Bring your own external authorization service to protect requests that go through your Gateway. 
+---
+
+Bring your own external authorization service to protect requests that go through your Gateway. 
+
+## About external auth {#about}
+
+{{< reuse "docs/snippets/product-name-caps.md" >}} lets you integrate your own external authorization service to your Gateway, based on the [Envoy external authorization filter](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/security/ext_authz_filter). Then, this external authorization service makes authorization decisions for requests that go through the Gateway, as shown in the following diagram.
+
+```mermaid
+sequenceDiagram
+    participant Client
+    participant Gateway
+    participant Ext Auth Service
+    participant Backend
+
+    Client->>Gateway: 1. HTTP Request
+    Gateway->>Ext Auth Service: 2. Authorization Request
+    Ext Auth Service-->>Gateway: 3. Authorization Decision
+    
+    alt Authorized
+        Gateway->>Backend: 4. Forward Request
+        Backend-->>Gateway: Response
+        Gateway-->>Client: Response
+    else Not Authorized
+        Gateway-->>Client: 5. 403 Forbidden
+    end
+```
+
+1. The Client sends a request to the Gateway.
+2. The Gateway forwards the request to the Ext Auth Service.
+3. The Ext Auth Service makes a decision as to whether the request is authorized, based on headers, parameters, or other credentials.
+4. If authorized, the Gateway forwards the request to the Backend app, which then sends back a response to the Client through the Gateway.
+5. If not authorized, the Gateway rejects the request and by default returns a 403 Forbidden response to the Client.
```

Comment:
wouldn't it be 401 for authentication and 403 for unauthorized requests

Reply:
It is 403 by default